src/layman/upgrade/upgrade_v2_0.py

```diff
@@ -274,6 +275,17 @@ def migrate_layers():
             'slugified_time_regex_format': slugified_time_regex_format,
         }
 
+        # Move input files
+        logger.info("      moving input files")
+        new_path = layer_file_util.get_layer_dir(layer_uuid)
+        input_file.ensure_layer_input_file_dir(layer_uuid)
+        input_files = util.get_layer_input_files(workspace, layername)
+        name_input_file_by_layer = image_mosaic is None or input_files.is_one_archive
+        for filename in input_files.raw_or_archived_paths:
+            file_name_ext = f"{layer_uuid}.{os.path.splitext(filename)[1]}" if name_input_file_by_layer else os.path.basename(filename)
+            dst_path = os.path.join(new_path, 'input_file', file_name_ext)
+            shutil.move(filename, dst_path)
+
```

Comment:
I suggest to carefully delete the original `input_file` directory with `os.rmdir` just after moving input files.

Reply:
Incorporated.